website/static/js/share/results.js

```diff
@@ -1,126 +1,280 @@
+            });
+
+        return m('span.pull-left', {style: {'text-align': 'left'}},
+            ctrl.showAll || result.contributors.length < 8 ?
+                contributorViews :
+                m('span', [
+                    contributorViews.slice(0, 7),
+                    m('br'),
+                    m('a', {onclick: function(){ctrl.showAll = !ctrl.showAll;}}, 'See All')
+                ])
+        );
+
+    }
+};
+
+var Contributor = {
+    view: function(ctrl, params) {
+        var contributor = params.contributor;
+        var index = params.index;
+        var vm = params.vm;
+        return m('span', [
+            m('span', index !== 0 ? ' · ' : ''),
+            m('a', {
+                href: '#',
+                onclick: function() {
+                    utils.updateFilter(vm, 'match:contributors.familyName:' + contributor.familyName, true);
+                    utils.updateFilter(vm, 'match:contributors.givenName:' + contributor.givenName, true);
+                }
+            }, contributor.name)
+        ]);
+    }
+};
+
+var Subjects = {
+    controller: function(vm) {
+        var self = this;
+        self.showAll = false;
+    },
+    view: function(ctrl, params){
+        var result = params.result;
+        var subjectViews = $.map(result.subjects || [], function(subject, i) {
+            return m.component(Subject, $.extend({subject: subject}, params));
+        });
+        if (ctrl.showAll || (result.subjects || []).length <= 5) {
+            return m('span', subjectViews);
+        }
+        return m('span', [
+            subjectViews.slice(0, 5),
+            m('br'),
+            m('div', m('a', {href: '#', onclick: function() {ctrl.showAll = !ctrl.showAll;}},'See All'))
+        ]);
+
+    }
+};
+
+var Subject = {
+    view: function(ctrl, params) {
+        var subject = params.subject;
+        var vm = params.vm;
+        return m('span', m('a.badge.pointer', {href: '#', onclick: function(){
+                utils.updateFilter(vm, 'match:subjects:"' + subject + '"', true);
+            }}, $.truncate(subject, {length: 50}), ' '
+        ));
+    }
+};
+
+var Footer = {
+    controller: function(params) {
+        var self = this;
+        this.result = params.result;
+        this.cleanResult = m.prop(null);
+        this.missingError = m.prop(null);
+        this.showRawNormalized = m.prop(false);
+        this.loadRawNormalized = function() {
+            utils.loadRawNormalized(this.result).then(
+                function(cleanData) {
+                    self.cleanResult(cleanData);
+                }, function(error) {
+                    self.missingError(error);
+                }
+            );
+        };
+    },
+    view: function(ctrl, params) {
+        var result = params.result;
+        var vm = params.vm;
+        return m('div', [
+            m('span',
+                'Released on ' + new $osf.FormattableDate(result.providerUpdatedDateTime).local,
```

Comment:
This text should probably be muted (use `.text-muted`), as it's supplementary information.